routes/api.js

```diff
@@ -288,13 +288,14 @@ var getMembersBySkills = async function (req, res, next) {
  * @apiUse OnNotFoundError
  */
 var addMember = function (req, res, next) {
+    console.log('req body', req.body);
     try {
-        if (req.body.hasOwnProperty("memberProfile")) {
-            let newMember = req.body.memberProfile;
```

Comment:
This part is required to evaluate whether the request is well formed JSON.

Reply:
req.body.hasOwnProperty('memberProfile'); kept coming up with an error. I believe it kept saying it is not a function.

Reply:
we can change it to <code> 'memberProfile' in req.body</code> . The reason this happens is because hasOwnProperty is implementation dependent and doesn't work well in all environments.  So this could be as simple as you and me having different version of Node.js. However <code>Object.Prototype.hasOwnProperty </code> is inherited from ES6 (JavaScript) and is same whereever you run.  But a fail safe way is to run the check as I've specified above.